app/sims/__init__.py

```diff
@@ -162,11 +163,19 @@ def one_time_simulation():
         sim.run()
         sim.config.decompose_spectrum = decompose
     except Exception as e:
-        return [f"SimulationError: {e}", True, no_update]
+        return [f"SimulationError: {e}", True, no_update, no_update]
 
     process_data = mrsim_data["signal_processors"]
     for proc, mth in zip(process_data, sim.methods):
         processor = SignalProcessor.parse_dict_with_units(proc)
+
+        # Adjust baseline offset for multi-spin_system spectra
+        # Otherwise given baseline offset will be multiplied by number of spin_systems
+        # (future) need to adjust polynomial as well when implemented
+        for op in processor.operations:
```

Comment:
This piece of code compensates for the implementation baseline offset within the mrsimulator library. Once a implementation is decided this should be removed.